org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/thingTranslator.js

```diff
@@ -101,8 +109,8 @@ var deviceId;
 var deviceType = 'thermostats';
 var winkHelper;
 
-// This translator class implements the 'org.opent2t.sample.thermostat.superpopular' interface.
-class WinkThermostat {
+// This translator class implements the 'org.opent2t.sample.thermostat.superpopular' schema.
+class Translator {
```

Comment:
This was a magic string, which was not gleaned from the schema. We need all strings in the translator (e.g. the method names) to be gleaned from the schema so we can auto-generate translator stubs e.g. in the yeoman generator. Otherwise we will need to ask the user about what they want to name the class.... since it is not used anywhere that seemed unnecessary to me.